### ISSUE TITLE
Why set shuffle = False, when using distributed-training?

### ISSUE THREAD

**Issue by zhangming8** [NVIDIA/tacotron2#396]
Hi, thanks for your greate repo. Could you tell me the reason: 
(1). why set shuffle=False when using distributed-training?
(2). can I set num_workers>1 ? 

The code is:
train_sampler = DistributedSampler(trainset) 
 shuffle = False
train_loader = DataLoader(trainset, num_workers=1, shuffle=shuffle, sampler=train_sampler, batch_size=args.batch_size, pin_memory=False, drop_last=True, collate_fn=collate_fn)
